components/playlist/browser/resources/components/playerSeeker.tsx

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import styled from 'styled-components'
+import { color, font } from '@brave/leo/tokens/css'
+
+interface Props {
+  videoElementRef: React.RefObject<HTMLVideoElement>
+}
+
+const SeekerContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const TimeContainer = styled.div`
+  display: grid;
+  grid-template-columns: auto auto;
+  justify-content: space-between;
+
+  color: ${color.text.secondary};
+  font: ${font.primary.xSmall.regular};
+`
+
+const StyledProgress = styled.progress`
+  --progress-background: #423eee; /* TODO(sko) Neither color.button.background or color.semantic.button.background exist for now */
+  --progress-bar-height: 12px;
+  --progress-stroke-thickness: 2px;
+
+  width: 100%;
+  height: var(--progress-bar-height);
+  appearance: none;
+  position: relative;
+
+  &::-webkit-progress-bar {
+    background: rgba(0, 0, 0, 0.1);
+    height: var(--progress-stroke-thickness);
+    border-radius: calc(var(--progress-stroke-thickness) * 0.5);
+    opacity: 1;
```

Comment:
Probably don't need to set the opacity to 1? But not 100% sure :laughing: With CSS it could be anything :D